src/status_im2/contexts/onboarding/common/navigation_bar/view.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im2.contexts.onboarding.common.navigation-bar.view
+  (:require [quo2.core :as quo]
+            [react-native.core :as rn]
+            [status-im2.contexts.onboarding.common.navigation-bar.style :as style]
+            [utils.re-frame :as rf]))
+
+(defn navigation-bar
+  [{:keys [on-press-info]}]
+  [rn/view {:style style/navigation-bar}
+   [quo/page-nav
```

Comment:
could we just set style to page-nav ? so we don't have extra view

Reply:
Yes, done